db/migrate/20220913150929_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
```

Comment:
- [ ] coments and likes will be created on a post. you are missing the counters
eg
```ruby
    t.integer :comments_counter
    t.integer :likes_counter
```